docs/articles/blog/2019-2-7-testcafe-v1-0-0-released.md

```diff
@@ -0,0 +1,162 @@
+---
+layout: post
+title: TestCafe v1.0.0 Released
+permalink: /blog/:title.html
+---
+# TestCafe v1.0.0 Released
+
+Meet TestCafe v1.0.0, our first major update that includes some big features: video recording, configuration file, "live mode" for rapid test development and much more.
+
+<!--more-->
+
+## Breaking Changes
+
+TestCafe v1.0.0 introduces minor changes to the behavior and programming interface. You can find the details in the [migration guide](https://devexpress.github.io/testcafe/blog/migration-from-testcafe-v0-x-y-to-v1-0-0.html).
```

Comment:
The behavior of what?

Reply:
(the same response as in the previous case)